src/app/utils/formatAge.ts

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
Didn't find any library to easily format the age, so written this utility function with a unit test.